react-frontend/src/components/StyleComponents/card.js

```diff
@@ -194,7 +194,6 @@ export const CommunityCardStyled = styled(Link).attrs({
   className: 'cardRound',
 })`
   background: #003366;
-  border: 5px solid rgb(242, 242, 242);
```

Comment:
This, in combination with styles already established on other parts of the card, were causing:
![Screen Shot 2021-06-15 at 10 13 27 AM](https://user-images.githubusercontent.com/16946297/122095979-e12b6880-cdc2-11eb-86f7-b86c2ffcd398.png)